src/Models/NodesInterface.php

```
@@ -0,0 +1,39 @@
+
+    /**
+     * Get the current node in the collection.
+     *
+     * @return null|NodeInterface
+     */
+    public function current(): ?NodeInterface;
+
+    /**
+     * @return NodesShape
+     */
+    public function jsonSerialize(): array;
```

PHPStan (2.1.15)

Method OpenFGA\Models\NodesInterface::jsonSerialize() has invalid return type OpenFGA\Models\NodeShape.


(class.notFound)